project_app/components/AdminLogin.js

```diff
@@ -0,0 +1,185 @@
+import { useState, useContext, useEffect } from "react";
+import { GlobalStateContext } from '../pages/_app';
+import styled from "styled-components";
+//import styles from "../styles/AdminLogin.module.css";
+import { CFormInput, CFormFloating, CFormLabel } from "@coreui/react";
+
+import "@coreui/coreui/dist/css/coreui.min.css";
+
+import Swal from "sweetalert2";
+
+import { useRouter } from 'next/router';
+
+export default function AdminLogin() {
+    const router = useRouter();
+    const { token, setToken } = useContext(GlobalStateContext);
+    const [emailOrUsername, setEmailOrUsername] = useState("");
+    const [password, setPassword] = useState("");
+
+    useEffect(() => {
+        console.log(token);
+        if (token != null) {
+            // Verify that token is not tampered with using api
+            router.push('adminpage');
```

Comment:
what is the diff of admin and adminpage?

Reply:
Admin is the admin login page and admin page will be what an admin sees after loggin in